style.css

```diff
@@ -0,0 +1,294 @@
+  font-size: 16px;
+  line-height: 32px;
+}
+
+button {
+  margin-top: 16px;
+  padding: 16px 48px;
+  font-size: 16px;
+  border: none;
+  color: #fdcc04;
+  background: #000;
+  cursor: pointer;
+  border-radius: 50px;
+}
+
+button:hover {
+  background: #fff;
+  color: #000;
+}
+
+.column-right {
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+  padding: 0px 32px;
+}
+
+.image {
+  width: 100%;
+  height: 100%;
+}
+
+
+
+.demo-title {
+  text-align: center;
+  font-size: 1,5rem;
+  margin-bottom: 25px;
+}
+
+.table {
+  display: table;
+  width: 100%;
+}
+.table .option {
+  box-shadow: 3px 3px 30px 5px rgba(204,204,204,0.9);
+  width: 45%;
+  background: #fdcc04;
+  float: left;
+  padding: 4%;
+  text-align: center;
+  -webkit-transition: all .3s ease-in-out;
+  transition: all .3s ease-in-out;
+}
+.table .option:nth-child(even) {
+  margin: 0 2%;
+}
+.table .option:hover {
+  cursor: pointer;
+  box-shadow: 0px 2px 30px rgba(0, 0, 0, 0.3);
+  -webkit-transform: scale(1.04);
+          transform: scale(1.04);
+}
+
+.table .option:hover .front {
+  opacity: 0;
+  visibility: hidden;
+}
+.table .option:hover .back {
+  opacity: 1 !important;
+  visibility: visible !important;
+}
+.table .option:hover .back a.button {
+  -webkit-transform: translateY(0px) !important;
+          transform: translateY(0px) !important;
+}
+.table .option hr {
+  border: none;
+  border-bottom: 1px solid #04040457;
+}
+.table .option i {
+  font-size: 20px;
+  color: #0c0c0c;
+  -webkit-transition: all .3s ease-in-out;
+  transition: all .3s ease-in-out;
+}
+.table .option h1 {
+  margin: 10px 0;
+  color: #212121;
+  -webkit-transition: all .3s ease-in-out;
+  transition: all .3s ease-in-out;
+}
+.table .option p {
+  color: #000000;
+  padding: 0 10px;
+  line-height: 1.3;
+}
+.table .option .price {
+  position: relative;
+}
+.table .option .front span.price {
+  font-size: 1rem;
+  text-transform: uppercase;
+  margin-top: 20px;
+  display: block;
+  font-weight: 700;
+  position: relative;
+}
+.table .option .price .front span.price b {
+  position: absolute;
+  font-size: 10px;
+  margin-left: 2px;
+  font-weight: 600;
+}
+.table .option .price .back {
+  opacity: 0;
+  visibility: hidden;
+  -webkit-transition: all .3s ease-in-out;
+  transition: all .3s ease-in-out;
+}
+.table .option .price .back a.button {
+  background: #bd0010;
+  padding: 15px 20px;
+  display: inline-block;
+  text-decoration: none;
+  color: white;
+  position: absolute;
+  font-size: 13px;
+  top: -5px;
+  left: 0;
+  right: 0;
+  width: 150px;
+  margin: auto;
+  text-transform: uppercase;
+  -webkit-transform: translateY(20px);
+          transform: translateY(20px);
+  -webkit-transition: all .3s ease-in-out;
+  transition: all .3s ease-in-out;
+}
+.table .option .price .back a.button:hover {
+  background: #9c020f;
+}
+
+.ramka
+{
+
+background-color: rgba(243, 239, 239, 0.475);
+width: 230px;
+padding: 15px;
+margin-left: 10px;
```

Comment:
jak ustawiasz wszedzie margin to lepiej uzywac skladni margin: top right down left czyli np margin: 20px auto 20px 10px